install.md

```diff
@@ -83,8 +83,8 @@ xUbuntu 18.04
 To install, choose a supported version for your operating system, and export it
 as a variable, like so: `export VERSION=1.19`
 
-We also save releases as subprojects. If you'd, for instance, like to use `1.19.1`
-you can set `export VERSION=1.19:1.19.1`
+We also save releases as subprojects. If you'd, for instance, like to use `1.24.5`
+you can set `export VERSION=1.24.5` and `export SUBVERSION=$(echo $VERSION | awk -F'.' '{print $1"."$2}')`
```

Comment:
@FerminCastro since it's markdown, it won't make a new line break if you just put it all on a newline:
```
we also save releases as subprojects. If you'd, for instance, like to use `1.24.5`
you can set `export VERSION=1.24.5` and
`export SUBVERSION=$(echo $VERSION | awk -F'.' '{print $1"."$2}')`
```
should work